go/ops/local_publisher.go

```diff
@@ -53,7 +53,13 @@ func (*LocalPublisher) PublishLog(log Log) {
 	logrus.StandardLogger().WithFields(fields).Log(level, log.Message)
 }
 
-/// logrusLogLevel maps the current Level of the logrus logger into a pf.LogLevel.
+// PublishStats implements Publisher
+func (*LocalPublisher) PublishStats(event StatsEvent) {
+	// TODO: will this ever get called?
+	logrus.WithField("stats", event).Error("got local stats event")
+}
+
+// / logrusLogLevel maps the current Level of the logrus logger into a pf.LogLevel.
```

Comment:
There's an extra `/` in front of this comment. The comment itself seems pretty redundant with the self-explanatory code also.